### ISSUE TITLE
Another user is using this camera

### ISSUE THREAD

**Issue by thales-man** [brutella/hkcam#61]
This message happens after the first use from the home app. The camera led is on which would suggest something is not being released after first use.

I’ve done several installs and the result is always the same.

PI 2, PI v1.3 camera, Buster, fresh install and patched to date. 
Ansible deployment from Mac.
IOS 12.4.1

**Comment by brutella** [comment]
HomeKit should automatically disconnect once the user dismisses the live stream in the Home app.
You may want to try using a different HomeKit app to access the camera stream. 

Maybe there is a bug in Apple's Home app, which keeps the live stream running forever?

**Comment by thales-man** [comment]
you may be right; sadly, that's what you have to work with.

**Comment by thales-man** [comment]
I should add if it doesn't work with Apples home kit app, it isn't HomeKit compatible...

**Comment by jschonenberg** [comment]
I'm running into the same issue. Opening the live feed just after a reboot works fine. After that the live feed appears to remain locked, which means the error "Wait until someone else in this home stops viewing this camera" won't go away anymore.

**Comment by brutella** [comment]
I think the issue is related to Raspbian Buster – it works fine when running Raspbian Stretch.
I currently don't have time to look into the issue of supporting Buster.

**Comment by jschonenberg** [comment]
Thanks for the reply, that's valuable information! Using Raspbian Stretch is an acceptable workaround for me. I'll give it a try soon and update here with the results.

**Comment by SegaBoy64** [comment]
I've had these issues on Stretch since day 1 - so I'm not convinced that is the problem.

I couldn't get to the bottom of it and snow simply run a cron job every hour to restart the hkcam process; it's ugly but it does alleviate the issue.

**Comment by alexsalex** [comment]
Same issue:

```
pi@hkcam-entrance:~ $ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 9.9 (stretch)
Release:	9.9
Codename:	stretch
pi@hkcam-entrance:~ $ uname -a
Linux hkcam-entrance 4.19.42+ #1219 Tue May 14 21:16:38 BST 2019 armv6l GNU/Linux
pi@hkcam-entrance:~ $
```

**Comment by PvdGulik** [comment]
This solves the problem instead of rebooting the raspberry pi.

`pi@HKCam:~ $ sudo service hkcam stop
Warning: The unit file, source configuration file or drop-ins of hkcam.service changed on disk. Run 'systemctl daemon-reload' to reload units.
pi@HKCam:~ $ sudo systemctl daemon-reload
pi@HKCam:~ $ sudo service hkcam stop
pi@HKCam:~ $ sudo service hkcam start`

So it looks this is de cause "The unit file, source configuration file or drop-ins of hkcam.service changed on disk."

Maybe it helps you to find the solution

**Comment by brutella** [comment]
I've just released version [0.1.0](https://github.com/brutella/hkcam/releases/tag/v0.1.0), which should run much more stable now. The instructions in the readme and the ansible script work for Raspberry Pi OS.
